### PR TITLE
Change tmp dir

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -33,16 +24,16 @@ def ignored_warning(warning):
     finally:
         set_ignored_warning(warning, False)
 
+
 @contextmanager
-def _tmp_file(content, extension):
-    with tempfile.NamedTemporaryFile(suffix='.' + extension, mode='w') as tmp_file:
+def _tmp_file(tmp_folder, content, extension):
+    suffix = '.' + extension
+    with tempfile.NamedTemporaryFile(suffix=suffix,
```
Comment (asanin-epfl) [inline tests/utils.py:31]: Can you please make a comment here why this logic is chosen? So no one tries to optimize this in the future.

Comment (mgeplf) [reply tests/utils.py:31]: which logic?

Comment (asanin-epfl) [reply tests/utils.py:31]: that this is required for Windows, no?

Comment (asanin-epfl) [reply tests/utils.py:37]: thank you

**Review by asanin-epfl — APPROVED** [review]

File: tests/utils.py (hunk 1 above)
Comment (mgeplf) [reply tests/utils.py:37]: np, thanks for the review

**Pull request merged**